src/services/yoga-logger.js

```diff
@@ -3,34 +3,57 @@ import { addDays, format } from "date-fns";
 
 import { getXataClient } from "./../xata.js";
 
-export function truncate(str, maxLength) {
+const ALLOWED_CHANNELS = ["yoga", "exercise"];
+
+function truncate(str, maxLength) {
   if (str.length > maxLength) {
     return str.substring(0, maxLength) + "...";
   }
   return str;
 }
 
-export function allowedMessage(message) {
+// Calculate the number of days with yoga sessions
+async function daysOfYoga(xata, discordUserId, sessionDateString) {
+  const result = await xata.db.session.aggregate(
```

Comment:
This does not work when I added the exercise: "yoga", it will not give me all my previous exercises in the beta database :/ Do you know what might be wrong?

Reply:
I've been playing with it in the playground, and it's weird how it goes from 37 days to 3 days when adding exercise: "yoga".

Reply:
Exactly. I guess we can move the check from database to JavaScript instead. Maybe we could calculate the count and group by exercise?

Reply:
It was a bug, I think they fixed it.